capstone-backend/src/main/java/com/google/sps/servlets/ClubSearchServlet.java

```diff
@@ -0,0 +1,58 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Map;
```

Comment:
Map is never used.

Reply:
The ClubSearchServlet class was merged with the ClubServlet and this line has been removed.